src/main/java/io/seqera/tower/cli/commands/pipelines/AbstractPipelinesCmd.java

```diff
@@ -37,7 +37,7 @@ public AbstractPipelinesCmd() {
 
     protected PipelineDbDto pipelineByName(Long workspaceId, String name) throws ApiException {
 
-        ListPipelinesResponse list = api().listPipelines(Collections.emptyList(), workspaceId, null, null, name, "all");
+        ListPipelinesResponse list = api().listPipelines(Collections.emptyList(), workspaceId, null, null, "\"" + name + "\"", "all");
```

Comment:
Better check that is not already quoted and escape any other quote.

Here a ChatGPT proposal:
```java
public class StringQuoter {
    public static String quoteString(String input) {
        if (input == null) {
            return "\"\"";
        }

        // Escaping intermediate quotes
        String escapedInput = input.replace("\"", "\\\"");

        // Check if the string is already quoted
        if (escapedInput.startsWith("\"") && escapedInput.endsWith("\"")) {
            return escapedInput;
        } else {
            // Adding quotes
            return "\"" + escapedInput + "\"";
        }
    }

    public static void main(String[] args) {
        // Test the function
        String testString = "Hello, \"world\"!";
        System.out.println(quoteString(testString)); // Should print: "Hello, \"world\"!"

        String alreadyQuoted = "\"Hello, world!\"";
        System.out.println(quoteString(alreadyQuoted)); // Should print the same string without changes
    }
}
```